lua/lspconfig/configs/lua_ls.lua

```diff
@@ -48,7 +48,7 @@ require'lspconfig'.lua_ls.setup {
   on_init = function(client)
     if client.workspace_folders then
       local path = client.workspace_folders[1].name
-      if vim.loop.fs_stat(path..'/.luarc.json') or vim.loop.fs_stat(path..'/.luarc.jsonc') then
+      if path ~= vim.fn.stdpath('config') and (vim.loop.fs_stat(path..'/.luarc.json') or vim.loop.fs_stat(path..'/.luarc.jsonc')) then
```

Comment:
the PR title says `if ~/.config/nvim/.luarc.json exists` but here you are comparing `path ~= vim.fn.stdpath('config')` ("NOT equal to ~/.config/nvim") ... ?

Reply:
the PR says "DON'T" skip the runtime if this directory exists. if this condition is taken, then the code does an early return, and the runtime is never loaded. so we want the condition to be false when the directory exists, hence ~= instead of ==.